.gitignore

```diff
@@ -1,6 +1,158 @@
-composer.phar
-composer.lock
+# Behat
+/features/fixtures/TestApp/cache/
+
+# Created by https://www.gitignore.io
+
+### Symfony ###
+# Cache and logs (Symfony2)
+/app/cache/*
+/app/logs/*
+!app/cache/.gitkeep
+!app/logs/.gitkeep
+
+# Cache and logs (Symfony3)
+/var/cache/*
+/var/logs/*
+!var/cache/.gitkeep
+!var/logs/.gitkeep
+
+# Parameters
+/app/config/parameters.yml
+/app/config/parameters.ini
+
+# Managed by Composer
+/app/bootstrap.php.cache
+/var/bootstrap.php.cache
+/bin/*
+!bin/console
+!bin/symfony_requirements
 /vendor/
-/features/fixtures/TestApp/cache/*
-/features/fixtures/TestApp/logs/*
+
+# Assets and user uploads
+/web/bundles/
+/web/uploads/
+
+# PHPUnit
+/app/phpunit.xml
+/phpunit.xml
+
+# Build data
+/build/
+
+# Composer PHAR
+/composer.phar
+
+### Symfony Patch ###
+/web/bundles/
+/web/uploads/
+/web/css/
+/web/js/
+
+
+### Intellij ###
```

Comment:
Yes. Please let is as-is and remove `composer.lock` as this is a library and not a standalone project.